solr/solrj/src/java/org/apache/solr/common/params/CollectionParams.java

```diff
@@ -70,6 +70,19 @@ public boolean isHigherOrEqual(LockLevel that) {
     }
   }
 
+  /**
+   * <p>(Mostly) Collection API actions that can be sent by nodes to the Overseer over the <code>/overseer/collection-queue-work</code>
+   * ZooKeeper queue.</p>
+   *
+   * <p>Some of these actions are also used over the cluster state update queue at <code>/overseer/queue</code>, and really
```

Comment:
While I agree with the point you're making, this reads a bit snarky or sarcastic. Maybe leave the suggested improvement as a `TODO`?